admin-dev/themes/new-theme/js/app/utils/init-components.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import TranslatableField from '@js/components/translatable-field.js';
+import TranslatableInput from '@js/components/translatable-input.js';
+import TinyMCEEditor from '@js/components/tinymce-editor.js';
+import TaggableField from '@js/components/taggable-field.js';
+
+const initPrestashopComponents = () => {
+  window.prestashop = {...window.prestashop};
+  window.prestashop.instance = {...window.prestashop.instance};
+
+  window.prestashop.component = {
+    initPrestashopComponents(components) {
```

Comment:
```suggestion
    initComponents(components) {
```

This way the "public" API respects the ADR naming https://github.com/PrestaShop/ADR/blob/master/0009-expose-js-components-using-window-variable.md#decision

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/20591/commits/4a1efe9ddf00bcff6588b8c06560ec3a1dc08164 and https://github.com/PrestaShop/PrestaShop/pull/20591/commits/458ca9b6edb4361db4cac8fed10bc3c96f155209